components/docs-chef-io/content/automate/ha_remove_single_node_from_cluster.md

```diff
@@ -107,4 +121,18 @@ The command requires some arguments to determine the type of node you want to re
     chef-automate node remove --postgresql-ip "<postgresql-ip-address>"
     ```
 
+You can remove other unreachable nodes while removing a node
+
+- To remove unreachable nodes while removing a node, pass the flag `--remove-unreachable-node` or `-r` like example below
```

Comment:
- To remove unreachable nodes while removing a node, pass the flag `--remove-unreachable-node` or `-r` like the example below: